CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing
+
+To develop on this project, please fork the repo and clone into your `$GOPATH`.
+
+Dependencies are **not** checked in so please download those separately.
+Download the dependencies using [`dep`](https://github.com/golang/dep).
+
+```bash
```

Comment:
How about a Makefile for this instead of instructions?

Reply:
Where would this Makefile live? Do you just mean a make step for installing dependencies?